### PR TITLE
Add port custom

### PR DESCRIPTION
**Pull request by jakspok**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (vchrombie) [inline .idea/.gitignore:1]: The `.idea` folder has to be removed

File: setup.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
       description="Drive GrimoireLab tools to produce a dashboard",
       long_description=long_description,
       long_description_content_type='text/markdown',
-      url="https://github.com/chaoss/grimoirelab-sirmordred",
+      url="https://github.com/jakspok/grimoirelab-sirmordred",
```
Comment (vchrombie) [inline setup.py:47]: This change has to be reverted

**Review by vchrombie — CHANGES_REQUESTED** [review]
Hi @jakspok, thanks for opening the PR.

Can you remove the `.idea` and other not relevant changes from the PR to proceed with the review.

You can just remove them and amend it to the same commit. Also, it would be great if you can squash the commits into one.

**Comment by jjmerchante** [comment]
The current version of grimoirelab uses SortingHat service instead of a database. This change is obsolete. Closing the PR

**Pull request closed without merging**